c7n/actions/invoke.py

```diff
@@ -11,14 +11,20 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+try:
+    from botocore.config import Config
+except ImportError:
+    from c7n.config import Bag as Config
```

Comment:
sigh, there's no good way to test this line, but its to avoid having a dependency on botocore for other providers.